.eslintrcforjs.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
   },
   rules: {
     'semi': 'error',
+    'max-len': 'off',
```

Comment:
Could you only turn off this rule in the folder that contains the generated code?

Reply:
@huningxin OK, I will update it, thanks.